Files/Interacts/Interaction.cs

```diff
@@ -138,9 +138,12 @@ public async void OpenDirectoryInTerminal(object sender, RoutedEventArgs e)
 
             if (App.Connection != null)
             {
-                var value = new ValueSet();
-                value.Add("Application", terminal.Path);
-                value.Add("Arguments", string.Format(terminal.Arguments, CurrentInstance.ViewModel.WorkingDirectory));
+                var value = new ValueSet
+                {
+                    { "WorkingDirectory", CurrentInstance.ViewModel.WorkingDirectory },
+                    { "Application", terminal.Path },
+                    { "Arguments", string.Format(terminal.Arguments, CurrentInstance.ViewModel.WorkingDirectory) }
```

Comment:
To fix Windows Terminal opening after app launch:
```suggestion
                    { "Arguments", string.Format(terminal.Arguments, InstanceTabsView.NormalizePath(CurrentInstance.ViewModel.WorkingDirectory)) }
```

Reply:
Done.
Also change default arguments for other terminals and tested ok.